tests/CMakeLists.txt

```diff
@@ -3,40 +3,40 @@ add_definitions(-DCSV_IO_NO_THREAD)
 
 add_executable(albatross_unit_tests
 EXCLUDE_FROM_ALL
-#test_call_trace.cc
-#test_crossvalidation.cc
-#test_concatenate.cc
-#test_core_dataset.cc
-#test_core_distribution.cc
-#test_core_model.cc
-#test_covariance_function.cc
-#test_covariance_functions.cc
-#test_csv_utils.cc
-#test_distance_metrics.cc
-#test_eigen_utils.cc
-#test_evaluate.cc
-##test_functional_model.cc
-#test_map_utils.cc
-#test_model_adapter.cc
-#test_models.cc
-##test_outlier.cc
-#test_prediction.cc
-#test_parameter_handling_mixin.cc
-#test_radial.cc
-#test_random_utils.cc
-##test_scaling_function.cc
-#test_serializable_ldlt.cc
-##test_serialize.cc
-#test_traits_cereal.cc
-test_traits_core.cc
-#test_traits_covariance_functions.cc
-##test_tune.cc
-##test_tuning_metrics.cc
+    test_call_trace.cc
+    test_core_dataset.cc
+    test_core_distribution.cc
+    test_core_model.cc
+    test_covariance_function.cc
+    test_covariance_functions.cc
+    test_cross_validation.cc
+    test_csv_utils.cc
+    test_distance_metrics.cc
+    test_eigen_utils.cc
+    test_evaluate.cc
+    test_indexing.cc
+    test_map_utils.cc
+    test_model_adapter.cc
+    test_models.cc
+    test_parameter_handling_mixin.cc
+    test_prediction.cc
+    test_radial.cc
+    test_random_utils.cc
+    test_ransac.cc
+    test_scaling_function.cc
+    test_serializable_ldlt.cc
+    test_serialize.cc
+    test_traits_cereal.cc
+    test_traits_core.cc
+    test_traits_covariance_functions.cc
+    test_traits_evaluation.cc
+    test_tune.cc
+    test_tuning_metrics.cc
 )
 
 add_dependencies(albatross_unit_tests
     albatross
-#    clang-format-all
+    clang-format-all
```

Comment:
This seems like an odd dependency to have. Are you sure about it?

Reply:
Nah, I'm not sold on it.  I find it nice to automatically have formatting run all the time, but I can achieve that other ways.

Reply:
In the past I've usually had things like this run as a git hook, but that's more easily done when it's a shell script and not part of CMake. Plus that approach is more difficult to enforce.

Reply:
Yeah, and in this case the unit tests enforce that formatting has been run, so I'll just remove the dependency.